core/templates/services/state-interaction-stats.service.ts

```diff
@@ -87,7 +78,7 @@ export class StateInteractionStatsService {
     return state.interaction.id === 'TextInput';
   }
 
-  private getReadableAnswerString(state: State, answer): string {
+  private getReadableAnswerString(state: State, answer: any): string {
```

Comment:
@nishantwrp do you prefer introducing "any" now or should we just remove the type like it was before?

Reply:
I think this can be declared as I mentioned above.

Reply:
Done.